### ISSUE TITLE
Client STILL can't show ( ͡° ͜ʖ ͡°) lenny face 

### ISSUE THREAD

**Issue by KuroiMatoO** [telegramdesktop/tdesktop#26635]
### Steps to reproduce

1. Copy-paste "( ͡° ͜ʖ ͡°)" in the input
2. Send it

### Expected behaviour

Your massage with a "( ͡° ͜ʖ ͡°)" sent successfully! 
Amazing!

### Actual behaviour

But in reality when I type I see this:

![image](https://github.com/telegramdesktop/tdesktop/assets/2721793/948e1237-ed4e-4909-a635-651c83b561da)

And when I send it I see this:

![image](https://github.com/telegramdesktop/tdesktop/assets/2721793/7f4b31eb-5c5c-4c04-b416-f9e8a82319fd)

Come one guys this important issue is almost 10 years old by now:  https://github.com/telegramdesktop/tdesktop/issues/505

### Operating system

Windows 8, Windows 10, Android 9

### Version of Telegram Desktop

4.8.10 x64

### Installation source

Static binary from official website

**Comment by ilya-fedin** [comment]
This is not an important issue and all the text rendering happens by Qt, not by tdesktop itself, that's why it's unlikely to change (as no one would spend time to debug this minor issue and then spend time on the very time-consuming process of reporting the issue to Qt)

**Comment by KuroiMatoO** [comment]
It's so over......

**Comment by jjolmo** [comment]
happening also in Linux Mint with cinnamon. No ( ͡° ͜ʖ ͡°) Who knows what other mighthy characters we would never see

**Comment by ilya-fedin** [comment]
Well, lots of characters (#6056). And from my observations Qt stopped rendering even more such characters since 6.5. So it's better to not to use exotic characters if that annoys you.

**Comment by hbh7** [comment]
Only stale because there's no fix, not because there's no issue

**Comment by soredake** [comment]
Not stale.

**Comment by KuroiMatoO** [comment]
No, no, the issue is still unresolved and as important as ever.

**Comment by ilya-fedin** [comment]
Sorry but this is impossible to fix in tdesktop code. Someone would have to report that to Qt as it's who does the text rendering.
